components/discord_bot/actions/delete-channel/delete-channel.mjs

```diff
@@ -0,0 +1,19 @@
+import common from "../common.mjs";
+
+export default {
+  ...common,
+  key: "discord_bot-delete-channel",
+  name: "Delete Channel",
+  description: "Delete a Channel.",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    ...common.props,
+  },
```

Comment:
No need to overwrite `props` here since no additional props are being added. You can leave out the `props` section here.